libs/examples/src/sidenav/sidenav-with-top-bar-navigation-example/sidenav-with-top-bar-navigation-example.html

```diff
@@ -0,0 +1,15 @@
+<dt-sidenav-container>
+  <dt-sidenav mode="side" opened #sidenav>
+    <dt-sidenav-header>
+      Sidenav title
+    </dt-sidenav-header>
+    Sidenav content
+  </dt-sidenav>
+  <dt-top-bar-navigation aria-label="Main navigation" class="nav">
+    <dt-top-bar-navigation-item *ngIf="!sidenav.opened" align="start">
+      <button class="open-sidenav-button" (click)="sidenav.open()">
```

Comment:
This should be implemented with the `dtTopBarAction` directive that provides the styling

```html
 <dt-top-bar-navigation-item align="start">
    <button (click)="sidenav.open()" dtTopBarAction>
      <dt-icon name="menu-hamburger"></dt-icon>
    </button>
  </dt-top-bar-navigation-item>
```